baseapp_notifications/migrations/0006_notification_insert_document_id_and_more.py

```
@@ -0,0 +1,71 @@
+                name="delete_document_id",
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    func="\n            DELETE FROM baseapp_core_documentid\n            WHERE\n                content_type_id = (SELECT id FROM django_content_type WHERE app_label = 'baseapp_notifications' AND model = 'notificationsetting')\n                AND object_id = OLD.id;\n            RETURN NULL;\n            ",
+                    hash="b7c53ad5a456f276224e6dd5d82775ae12f9ffef",
+                    operation="DELETE",
+                    pgid="pgtrigger_delete_document_id_70e43",
+                    table="baseapp_notifications_notificationsetting",
+                    when="AFTER",
+                ),
+            ),
+        ),
+    ]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)